website/docs/introduction/features.md

```diff
@@ -4,4 +4,82 @@ title: Why should you choose Litmus?
+
+## Advantages of Litmus 2.0
+
+A high-level feature overview of Litmus 2.0 are as follows
+
+- The Addition of [Chaos Workflow](../concepts/chaos-workflow) creation, Chaos experiments become building blocks of a Chaos Workflow, to allow users to create a larger chaos scenario using sequential or parallel experiment executions.
+- Addition of [ChaosCenter](../getting-started/resources#chaoscenter) where you can take advantage of all these features and a lot more
+  - Workflow Creation
+    <span style={{color: '#909191'}}>
+    <ul>
+    <li>From Templates, Custom Workflows from Scratch (using ChaosHubs), From pre-created YAMLs</li>
+    <li>Chaos Experiments Sequence Control (Parallel as well as Sequential steps creation)</li>
+    <li>Creation of either Singular or Cron Workflows as Schedules</li>
+    <li>Attaching priority to Chaos Experiments based on your use cases</li>
+    </ul>
+    </span>
+  - Users & Teams
+    <span style={{color: '#909191'}}>
+    <ul>
+    <li>Creation of Users with Role Based Access Control</li>
+    <li>Creating a Team of multiple Users</li>
+    <li>Authenticating Users</li>
+    </ul>
+    </span>
+  - Monitoring & Observability
+    <span style={{color: '#909191'}}>
+    <ul>
+    <li>Connect a Data Source (from any Agent) and monitor workflows</li>
+    <li>Visualize workflow run statistics and aggregated schedules</li>
+    <li>Compare two or more Workflows</li>
+    <li>Upload shared/downloadable dashboards available in the community</li>
+    <li>Edit queries, Tune dashboards to create a custom one from scratch</li>
+    <li>Monitor effect of chaos in real time with interleaved events and metrics from Prometheus Datasource</li>
+    </ul>
+    </span>
+  - Workflow Management
+    <span style={{color: '#909191'}}>
+    <ul>
+    <li>Rolling out automated changes using GitOps</li>
+    <li>Allowing image addition from custom image server (both public and private)</li>
+    <li>Measure and Analyse the Resilience Score of each workflow</li>
+    </ul>
+    </span>
+
+Litmus itself is composed of microservices. And we made sure that by adding the above features for 2.0, seamlessly integrates the additional microservices in conjunction with the existing one. Litmus 2.0 is completely backwards compatible. No features are deprecated.
+
+The migration path is about constructing new artifacts such as Chaos Workflows that include the current chaos experiments in use by the users.
+
+Below is a high level comparison between Litmus 1.x and Litmus 2.0 providing a holistic view of the feature additions you get in Litmus 2.0.
```

Comment:
Can we have a heading here like `Feature Comparison between 1.0 and 2.0` so that it shows on the right hand sidebar